librz/core/cmd/cmd_debug.c

```diff
@@ -3904,6 +3796,101 @@ RZ_IPI int rz_cmd_debug_continue_until(void *data, const char *input) {
 	return RZ_CMD_STATUS_OK;
 }
 
+// ds
+RZ_IPI RzCmdStatus rz_cmd_debug_step_handler(RzCore *core, int argc, const char **argv) {
```

Comment:
use `	CMD_CHECK_DEBUG_DEAD(core);`in all these handlers.

Reply:
no `CMD_CHECK_DEBUG_DEAD` ?

Reply:
damn... That means however that CMD_CHECK_DEBUG_DEAD breaks everything in esil, not just this command. We probably just don't have the tests for the other cases.

@XVilka @thestr4ng3r do you know how that ESIL thing work? Is it considered a dbg plugin? Or do we need to do an `if (debug) ... else if (esil) ....` every time?? :/

Reply:
It's quite complicated. AFAIR @thestr4ng3r did some decoupling of the ESIL from debug engine, but it's still tightly coupled. For example, see the `ar`/`dr` commands that share the code. Maybe we could think about similar approach in `aes`/`ds` as well. But it would be a big change on it's own. I suggest to open an issue to track this and do not touch this in the PR.